.prettierignore

```diff
@@ -0,0 +1,2 @@
+.github/
+package-lock.json
```

Comment:
can you also add ios/ and android/ in here? i think those are coming up with prettier errors but shouldn't be modified

Reply:
Good call -- updated!